.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+# Deploys to Digital Ocean
+# After: https://medium.com/swlh/how-to-deploy-your-application-to-digital-ocean-using-github-actions-and-save-up-on-ci-cd-costs-74b7315facc2
+name: Deploy & Run
+on:
+  push:
+    branches: [master]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy the app
+        uses: appleboy/ssh-action@v0.1.2
+        with:
+          host: ${{secrets.SSH_HOST}} # IP address of the server you wish to ssh into
+          key: ${{secrets.SSH_KEY}} # Public key of the server
+          username: ${{secrets.SSH_USERNAME}} # User of the server you want to ssh into
+
+          script: |
+            git clone https://github.com/OpenHospitalityNetwork/fedi-trustroots.git
+            echo 'Deployment successful to Digital Ocean'
+            cd fedi-trustroots
+            echo 'Installing dependencies...'
+            npm ci
+            echo 'Starting the app...'
+            npm start
```

Comment:
We current run these two commands in separate shells:

```
NODE_ENV=production npm run start:prod
NODE_ENV=production npm run webpack:server
```

I'm not sure if even `NODE_ENV=production npm start` which is simpler would do it. We also need to kill the current running processes.

Reply:
I removed running the app as for now, let's have checkout work first and then someone can try to figure out how to (re-)run it best.